examples/feature_demo/image_overlay.py

```diff
@@ -62,5 +62,5 @@
 controller = gfx.PanZoomController(camera, register_events=renderer)
 
 if __name__ == "__main__":
-    canvas.request_draw(lambda: renderer.render(scene, camera))
+    canvas.request_draw(lambda: gfx.render_with_logo(renderer, scene, camera))
```

Comment:
alright, this change i'm happy with. Should be "easy" to add to examples and whatnot without disruption.